socorro/processor/symbol_cache_manager.py

```diff
@@ -5,27 +5,31 @@
 import os
 import sys
 import tempfile
-import threading
-from datetime import datetime
-from collections import defaultdict
 from ordereddict import OrderedDict
 
 from configman import Namespace, RequiredConfig
-from configman.converters import class_converter, timedelta_converter
 
-from socorro.lib.datetimeutil import utc_now, UTC
-import pyinotify
 
+if os.uname()[0] == 'Darwin':
```

Comment:
You should make it != 'Linux', since this won't work on anything else (Windows, BSD, etc)